coderd/database/dbauthz/querier_test.go

```diff
@@ -282,11 +282,6 @@ func (s *MethodTestSuite) TestProvsionerJob() {
 		check.Args(database.UpdateProvisionerJobWithCancelByIDParams{ID: j.ID}).
```

Comment:
Note: most of the changes here are just moving the respective tests to `system_test.go` to keep things consistent.